examples/standard_platformer/lib/main.dart

```diff
@@ -9,40 +9,56 @@ import 'package:leap_standard_platformer/player.dart';
 import 'package:leap_standard_platformer/welcome_dialog.dart';
 
 void main() {
-  runApp(GameWidget(game: ExamplePlatformerLeapGame()));
+  runApp(
+    GameWidget(
+      game: ExamplePlatformerLeapGame(),
+    ),
+  );
 }
 
 class ExamplePlatformerLeapGame extends LeapGame
-    with HasTappables, HasKeyboardHandlerComponents {
+    with TapCallbacks, HasKeyboardHandlerComponents {
   late final Player player;
   late final SimpleCombinedInput input;
 
   @override
   Future<void> onLoad() async {
     await super.onLoad();
-
-    await loadWorldAndMap('map.tmx', 16);
-    setFixedViewportInTiles(32, 16);
```

Comment:
I think we need to find a way to include this with the new camera system. Something is fundamentally broken in this PR with the new camera, if you look at how this worked before (please run it locally with the old code) you'll see the background image fits the camera perfectly and moves with the player. Right now in this PR the background image, tiles for the map, and player sprite are all out of whack.

Reply:
See my comment on `loadWorldAndMap` I have a partial fix

Reply:
With my other suggestion, this is how I got this working again:

```dart
    await loadWorldAndMap(
      tileSize: 16,
      tiledMapPath: 'map.tmx',
      tileCameraWidth: 32,
      tileCameraHeight: 16,
    );

    input = SimpleCombinedInput();
    add(input);

    player = Player();
    cameraComponent.world!.add(player);
    cameraComponent.follow(player);
```